css/styles.css

```diff
@@ -0,0 +1,170 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightgrey;
+  font-family: sans-serif;
+  font-size: 19px;
+  margin-top: 0px;
+}
+
+.container {
+  width: 800px;
+  margin: 0 auto;
+}
+
+header {
+  background-color: black;
+  border-bottom: 4px solid #f8ff00;
+  padding: 0px 18px;
+}
+
+/*h1.title {
+  margin-bottom: 20px
+}*/
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+header a {
+  color: white;
+  margin-right: 12px;
+}
+
+nav {
+  text-align: right;
+  margin-bottom: 15px;
+}
+
+h1 {
+  color: #f8ff00;
+  padding: 15px 0px 0px 15px;
+  margin-top: 0px;
+  margin-bottom: 20px;
+  font-size: 30px;
+}
+
+h2 {
+  margin-bottom: 5px;
+}
+
+.uppercase {
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 20px;
+  font-family: Oswald !important;
+}
+
+.slogan {
+  background-color: #c6c6c6;
+  text-align: center;
+  font-style: italic;
+  padding: 5px;
+  font-size: 27px;
+  font-weight: bold;
+  font-family: Merriweather;
+}
+
+.strike {
+  color: darkgrey;
+  text-decoration: line-through;
+}
+
+.underline {
+  text-decoration: underline;
+  text-decoration-color: #f8ff00;
+}
+
+article {
+  background-image: url("../images/coworking.jpg");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 100%;
+  height: 400px;
+}
+
+aside {
+  height: 360px;
```

Comment:
@eddiejew try not to set height on containers, try to have the content dictate the height. Using margin and padding to apply spacing on the elements. 🔨